docs/building_and_running.md

```diff
@@ -1,5 +1,5 @@
 # Building BigWheels
-The recommended build system is CMake. This repo supports both in-tree, and out-of-tree builds.
+The recommended build system is CMake(version 3.20 and above). This repo supports both in-tree, and out-of-tree builds.
```

Comment:
nit: space before parenthesis > `The recommended build system is CMake (version 3.20 and above)`.

Reply:
Thanks for the review. This is done now.